### ISSUE TITLE
Using a different class in article mode

### ISSUE THREAD

**Issue by pzehner** [alex-ball/beamerswitch#18]
Hello, thank you for developing this convenient class!

According to the documentation, it seems that the `article` mode can only use the `article` class. It could be nice to be able to manually specify another class, like `IEEEtran`. Something in this spirit:

```latex
\documentclass[
  article,
  articleclass=IEEEtran,  % <-- here
  articleoptions={
    9pt,
    draft,
    conference,
    a4paper,
  },
  beameroptions={
    aspectratio=169,
    xcolor=table,
    ignorenonframetext,
  },
]{beamerswitch}
```

What do you think?

**Comment by alex-ball** [comment]
Great idea. I want to do a little more testing before packaging a release but I think commit 341f712 will do it.

**Comment by pzehner** [comment]
Awesome! I made some tests on my side with `IEEEtran`. The class is used, but the result is different from using the class directly. It seems that only the first argument of `articleoptions` is actually passed to the class. In the example case, I get in the logs:

```log
LaTeX Warning: Unused global option(s):    
    [ draft, conference, a4paper].         
```

Passing the options manually with `\PassOptionsToClass` works when used in the document file, but not when used in the class file (even hard coded).

To make it work, I have to comment my new lines:

```latex
  articleoptions={%
    9pt,%
    draft,%
    conference,%
    a4paper,%
  },
```

**Comment by alex-ball** [comment]
The only way I could fix this in beamerswitch would be to introduce code to strip spaces out of the `\cmdDC@beamerswitch@articleoptions` and `\cmdDC@beamerswitch@beameroptions` macros. It's doable but I'll have to see how efficiently it can be done (specifically, without too many new dependencies).

You have found two good workarounds in addition to simply writing it all on one line with no spaces:

```tex
articleoptions={9pt,draft,conference,a4paper}
```

**Comment by alex-ball** [comment]
This should be fixed in commit a3f8da4. Now released as v1.9.

**Comment by pzehner** [comment]
I tested it, it works perfectly! Thank you a lot!
